README.md

```diff
@@ -0,0 +1,20 @@
+# Carousel
+Displays images in a carousel.
+
+## Features
+* Set static images in the modeller
+
+## Dependencies
+Mendix 6: Tested on 6.8.0.
```

Comment:
Removed it.
But shouldn't being specific on the version save you from trouble in the event that the widget breaks in a lower or higher Mendix 6 (in this case) version?